librz/util/subprocess.c

```diff
@@ -1548,7 +1578,14 @@ RZ_API void rz_subprocess_pty_free(RZ_OWN RzPty *pty) {
 #endif
 
 RZ_API int rz_subprocess_ret(RzSubprocess *proc) {
-	return proc->ret;
+	if (proc->ret_sem) {
+		rz_th_sem_wait(proc->ret_sem);
+	}
```

Comment:
This should never happen. We should init the semaphore and it should be always be valid.

Reply:
Ok, semaphore is always valid in d2ad89da523df4a327b12367273e8658a4227f6e